test/test_dist_autograd_fork.py

```diff
@@ -1,21 +1,23 @@
 #!/usr/bin/env python3
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import process_group_dist_autograd_test
 import torch
-
-# dist_autograd_fork tests use double as the default dtype
-torch.set_default_dtype(torch.double)
-
-from dist_autograd_test import DistAutogradTest
 from common_distributed import MultiProcessTestCase
 from common_utils import run_tests
 
 
-class DistAutogradTestWithFork(MultiProcessTestCase, DistAutogradTest):
+# dist_autograd_fork tests use double as the default dtype
+torch.set_default_dtype(torch.double)
 
+
+class DistAutogradTestWithFork(
```

Comment:
Would I be correct if I assume this although this class name does not say process group but it will only run test using process group agent? If I add a new backend, I need to create two new classes: 1) `BackendXYZDistAutogradTest`, and 2) `BackendXYZDistAutogradTestWithFork `? 

If the above is true, shall we merge these two classes into one as you did in the first version, but highlight process group in the class name?

(Doesn't need to be addressed in this PR) And if this is just setting the fixture, I wonder whether we should do this using a decorator and keep `DistAutogradTestWithFork` class generic.

Reply:
The generic test suite to be re-used by other RPC bakcneds is `dist_autograd_test.DistAutogradTest`

I should rename this to be `ProcessGroupDistAutogradForkTest`.